src/commands/nfts/showNft.ts

```
@@ -176,16 +176,35 @@ export const showNft = async (
       )
     )
 
-    const price = getAssetRewardsFromDDOByService(ddo, 'nft-sales')
-      .getTotalPrice()
-      .div(10)
-      .multipliedBy(decimals)
+    try {
+      const price = getAssetRewardsFromDDOByService(ddo, 'nft-sales')
+        .getTotalPrice()
+        .div(10)
+        .multipliedBy(decimals)
+
+      logger.info(
+        chalk.dim(
+          `Price (NFT-1155): ${chalk.whiteBright(price)} ${chalk.whiteBright(
+            symbol
+          )}`
+        )
+      )
+    } catch {}
```

why do we need an empty catch?